src/Core/System/Resources/config/listing.xml

```diff
@@ -36,5 +36,14 @@
             <helpText>Products are marked as new, if their release date does not date back past the set number of days.</helpText>
             <helpText lang="de-DE">Produkte werden als neu markiert, wenn ihr Erscheinungsdatum nicht länger als die angegebene Anzahl an Tagen zurückliegt.</helpText>
         </input-field>
+
+        <input-field>
+            <name>productCountSteps</name>
+            <label>Selection of products per page</label>
+            <label lang="de-DE">Anzahlen der Produkte je Seite</label>
```

Comment:
There is nothing to pay. :-) Anzahl~en~

Reply:
Got it :D but I want to state there are multiple counts of products you can enter here. I checked it on duden.de and was confused as well but it is ok, somehow
![grafik](https://user-images.githubusercontent.com/1133593/89169884-24039980-d57f-11ea-8033-6b797b8dcb64.png)

Reply:
Oh, damn is that ugly 🙈

Reply:
I am still open for other suggestions though :D so I don't think this is perfect either.

Reply:
Hmmmm. In SW5 it's called `Auswahl Artikel pro Seite`. What do you think of `Produktauswahl pro Seite`?

Reply:
I am not d'accord with that in shopware 5 either. I am not even happy with the input field in the configuration :D 
Maybe @shopwareBot should comment on it.

Reply:
... or the guy who flagged this... @pweyck :-)

Reply:
Maybe add `DIe`?
`Die Anzahlen der Produkte...´

Reply:
![grafik](https://user-images.githubusercontent.com/1133593/89399261-82ad4c80-d712-11ea-9302-d663796af1c9.png)

Reply:
maybe we should ask @GitEvil about that

Reply:
I am afraid I do not get what you mean by "multiple counts of products". If you mean that there are multiple options to choose from (between 2 and 48 products per page), this does not justify the use of a plural form here. The number of products does not change, only the number of displayed products on one page, right? So its always just "eine bestimmte Anzahl an Produkten" und nur verschiedene bzw mehrere Möglichkeiten der Anzeige or Anzahlausprägungen so to say. I've never ever heard the word "Anzahlen" being used in the way suggested here. :)

Reply:
```suggestion
            <label lang="de-DE">Produkte pro Seite</label>
```